requests/models.py

```diff
@@ -420,8 +420,13 @@ def prepare_body(self, data, files, json=None):
         length = None
 
         if not data and json is not None:
+            # When urllib3 uses pyOpenSSL, it can only resume large uploads
```

Comment:
What does resuming large uploads have to do with urllib3 requiring a bytes object?